tt1-blocks/assets/css/blocks.css

```diff
@@ -67,10 +58,6 @@
 # Latest Posts
 --------------------------------------------------------------*/
 
-.wp-block-latest-posts {
-	padding-left: 0;
-}
-
 .wp-block-latest-posts.is-style-twentytwentyone-latest-posts-dividers {
```

Comment:
This sucks cause the style variation doesn't even look like the original theme. Can we get rid of it altogether and just make it into a block pattern?

Reply:
Yes, and it can always be adjusted in a later PR.